### PR TITLE
add two instanse and two security groups while using data and output

### PR DESCRIPTION
**Pull request by mohanedmoh**

### CODE REVIEW

File: Terraform/AWS/task5/data.tf [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
+}
+data "aws_availability_zones" "available" {
+  state = "available"
+}
```
Comment (geoffrey1330) [inline Terraform/AWS/task5/data.tf]: add newline at the end of file

File: Terraform/AWS/task5/instance.tf [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
+    Name = "${var.env_code}-allow_ssh-from-VPC"
+  }
+}
```
Comment (geoffrey1330) [inline Terraform/AWS/task5/instance.tf]: add newline at the end of file

File: Terraform/AWS/task5/output.tf [hunk 3]
```diff
@@ -0,0 +1,4 @@
+output "public-instance-IP" {
+  description = "IP of public instance"
+  value       = aws_instance.public.public_ip
+}
```
Comment (geoffrey1330) [inline Terraform/AWS/task5/output.tf]: add newline at the end of file

File: Terraform/AWS/task5/terraform.tfvars [hunk 4]
```diff
@@ -0,0 +1,3 @@
+env_code     = "Mentorship"
+public_cidr  = ["10.0.1.0/24", "10.0.2.0/24"]
+private_cidr = ["10.0.3.0/24", "10.0.4.0/24"]
```
Comment (geoffrey1330) [inline Terraform/AWS/task5/terraform.tfvars]: add newline at the end of file

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

**Comment by mohanedmoh** [comment]
Ops!!! sorry I forgot

File: Terraform/AWS/task5/instance.tf [hunk 5]
```diff
@@ -0,0 +1,69 @@
+resource "aws_instance" "public" {
```
Comment (geoffrey1330) [inline Terraform/AWS/task5/instance.tf:1]: created security group but didn't associate it with the instance

File: Terraform/AWS/task5/instance.tf [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+  instance_type = "t3.micro"
+  subnet_id     = aws_subnet.private[0].id
+  associate_public_ip_address = true
+  availability_zone = data.aws_availability_zones.available.names[0]
+  key_name = "ssh key"
+  tags = {
+    Name = "${var.env_code}-public"
+  }
+}
+resource "aws_instance" "private" {
```
Comment (geoffrey1330) [inline Terraform/AWS/task5/instance.tf:13]: created security group but didn't associate it with the instance

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: Terraform/AWS/task5/instance.tf (hunk 6 above)
Comment (mohanedmoh) [reply Terraform/AWS/task5/instance.tf:13]: I have this question, I tried to drop the first commit "assign and add new line...." but it is still here

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**